lib/std/debug.zig

```diff
@@ -66,6 +66,14 @@ pub fn print(comptime fmt: []const u8, args: anytype) void {
     nosuspend stderr.print(fmt, args) catch return;
 }
 
+/// Indicates code that is unfinshed. It will throw a compiler error in Release mode.
+pub fn todo(comptime desc: []const u8) noreturn {
+    if (builtin.mode != .Debug) {
+        @compileError("TODO: " ++ desc);
```

Comment:
There should probably be a way to allow `debug.todo` in Release binaries, following the accepted proposal. 
```suggestion
/// Indicates code that is unfinshed. It will throw a compiler error by default in Release mode. This behaviour can be controlled with `root.allow_todo_in_release`
pub fn todo(comptime desc: []const u8) noreturn {
    if (builtin.mode != .Debug and !(@hasDecl(root, "allow_todo_in_release") and root.allow_todo_in_release)) {
        @compileError("TODO: " ++ desc);
```